### DISCUSSION TITLE
Copilot suggesting variables and other things in new project from other projects

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19030] [Copilot Conversations]
Is it supposed to do this?

I was working with copilot in a financial application I wrote, and then started a new project to play around,
and it is suggesting datasets and variable names from the last project I was working on in this new project. 

It is kind of mucking up the usage of it on this new project with its seeming fixation on my last project. Is there a way for me to ask it to forget its recent suggestions so it stops doing this?

**Comment by ghost** [comment]
Anyone know about this?

**Reply by dv0zn3r** [reply]
I have noticed that if two projects are opened in IDE at the same time, but work is being done only in one, Copilot can from time to time suggest snippets from another opened project. When I closed another project, Copilot continued to suggest snippets from that project. Only after restarting the IDE with one project opened, suggestions changed to something else.
